Assets/Scripts/Interaction/Attack.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Enums;
+using Mirror;
+using Player.Body;
+using UnityEngine;
+
+namespace Interaction
+{
+    /// <summary>
+    /// Placeholder class to simulate attacks on other players. Inflicted damage will cause various effects
+    /// like bruising and bleeding, but they don't do anything yet. Damage can also sever bodyparts, which is also
+    /// only visual at the moment.
+    ///
+    /// Should be attached to player prefab.
+    ///
+    /// Mouse over other players (or yourself) and hit F to attack.
+    /// </summary>
+    public class Attack : NetworkBehaviour
+    {
+        [SerializeField] private GameObject attackParticleEffect;
+        [SerializeField] private AttackType attackType;
+        [SerializeField][Range(1,10)] private float damageAmount;
+
+        void Update()
+        {
+            CheckForAttack();
+        }
+
+        private void CheckForAttack()
+        {
+            if (!isLocalPlayer)
+            {
+                return;
+            }
+            
+            if (!Input.GetKeyDown(KeyCode.F))
+            {
+                return;
+            }
+
+            LayerMask layerMask = ~(1 << LayerMask.NameToLayer ("Player"));
+            Ray ray = Camera.main.ScreenPointToRay(Input.mousePosition);
```

Comment:
Do not use `Camera.main` in `Update()`

Reply:
Same as above, attack script is fake news